dev/arktype.io/docs/api/instanceof.md

```diff
@@ -4,8 +4,19 @@ hide_table_of_contents: true
 
 # instanceOf
 
-## text
+## operator
 
-```ts
-instanceOf: Ark["instanceOf"]
-```
+-   [instanceOf](./instanceof.md)
+
+## helper
```

Comment:
Can we make sure these are always specified in the same order?

Probably string, tuple, helper, omitting whatever isn't applicable